### ISSUE TITLE
Erekir core unit should be able to heal units. 

### ISSUE THREAD

**Issue by Cbao04** [Anuken/Mindustry-Suggestions#3577]
### Describe what you would like changed, and why.

Erekir is a hell of a planet so erekir core units should be able to heal units like it could heal blocks

### Describe the changes you want to propose. Include possible alternatives.

Since Erekir units have no support units Erekir's core unit should be able to heal erekir units.

**Comment by Mnemotechnician** [comment]
> i have no idea why this has been downvoted

this suggestion is clearly based on a skill issue; erekir sectors are already pretty easy, and an ability to heal your own units using just the core unit and nothing else would make them as easy as ground zero.

**Comment by OnyxFinger** [comment]
> > i have no idea why this has been downvoted
> 
> this suggestion is clearly based on a skill issue; erekir sectors are already pretty easy, and an ability to heal your own units using just the core unit and nothing else would make them as easy as ground zero.

Ok lemme point something out
You can heal walls, which is a stronger ability, and exactly the same mindset. How is healing units the problem ? In fact, I'd argue that if Erekir is easy, it's because the attacks are too weak, just like they are on Serpulo. 

The survival map's difficulty isn't very high, and sectors like 4 with a "second" base after you finished the first one doesn't send enough units during the second part of the game. During the first part, early units can be an issue when playing the map for the first time, because they're slightly annoying. They should be defended after like 3 waves though, so I think they're weak too.

Basically, that's not really the problem. - healing units is just some nice QoL-ish stuff -
